### PR TITLE
Edits to smooth and clarify language

### PR DESCRIPTION
**Pull request by GrafanaWriter**
Acting as a second set of eyes :)

### CODE REVIEW

File: docs/authoring.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
-# How to Write a Lineage
+# Tutorial on writing a lineage
 
-_It is strongly recommended to read the [Thema overview](overview.md) before beginning this tutorial._
+The most fundamental task in thema is writing a lineage. Writing a lineage is similar to other schema-writing systems: you're defining a specification for what data is supposed to look like, with two key differences:
 
-The first, most basic task to be done with thema is to write a lineage.
+1. CUE is the only language supported for schemas
+2. Schemas are wrutten _within_ a larger, well-defined structure - the lineage - which groups those schema together and enforces certain requirements.
```
Comment (malcolmholmes) [inline docs/authoring.md:6]: s/wrutten/written/

Comment (sdboyer) [inline docs/authoring.md:6]:
```suggestion
2. Schemas are written _within_ a larger, well-defined structure - the lineage - which groups those schema together and enforces certain requirements.
```

**Review by sdboyer — APPROVED** [review]
Amazing, these edits all look great. Thank you!

File: docs/authoring.md (hunk 1 above)
Comment (sdboyer) [reply docs/authoring.md]: good catch, ty

**Pull request merged**